community-operators/keycloak/csv.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: keycloak-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "Keycloak, KeycloakRealm, Kubernetes, Operator"
+    description: |-
+      A Kubernetes Operator based on the Operator SDK for syncing resources in Keycloak.
+    containerImage: quay.io/integreatly/keycloak-operator:master
+    createdAt: 2019-01-25-T20:00:00Z
+    support: integr8ly
+    alm-examples: |-
+      [{
+        "apiVersion": "aerogear.org/v1alpha1",
+        "kind": "Keycloak",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "adminCredentials": ""
+        }     
+      }]
+spec:
+  displayName: KeyCloak Operator
+  description: |-
+    A Kubernetes Operator based on the Operator SDK for syncing resources in Keycloak.
+  keywords: ['Keycloak', 'KeycloakRealm', 'operator', 'openshift', 'kubernetes']
+  version: 0.0.1
+  maintainers:
+  - name: integr8ly
+  provider:
+    name: integr8ly
+  labels:
+    name: keycloak-operator
+  selector:
+    matchLabels:
+      name: keycloak-operator
+  links:
+  - name: Keycloak Operator Source Code
+    url: https://github.com/integr8ly/keycloak-operator
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
```

Comment:
I thought we are allowing operators with `AllNamespaces` to `true` only. So should `OwnNamespace` and `SingleNamespace` be set to `false` as well?

Reply:
This isn't the case, we just want to be sure we only add operators that support `AllNamespaces` rather than remove support for other install modes in the CSV.